.travis.yml

```diff
@@ -1,5 +0,0 @@
-language: python
-python:
-  - "2.7"
-install: "pip install -r requirements.txt"
-script: ./check_links.sh
```

Comment:
1. All the redirect links are to fragment URL's, so they won't really get validated.
2. There's no index or link tree to crawl anymore, just pages for each URL that do not refer to one another.